.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "projects/ithor_rearrangement"]
```

Comment:
I didn't know about this,  but look awesome!

Reply:
Yah, having submodules is pretty neat. In theory this should let people have their own separate repositories while still allowing us to include them under the projects directory.